sql/mob_skill_lists.sql

```diff
@@ -713,10 +713,10 @@ INSERT INTO `mob_skill_lists` VALUES ('Kindred',169,559);
 INSERT INTO `mob_skill_lists` VALUES ('Kindred',169,560);
 INSERT INTO `mob_skill_lists` VALUES ('Kindred',169,563);
 INSERT INTO `mob_skill_lists` VALUES ('Kindred',169,1148);
-INSERT INTO `mob_skill_lists` VALUES ('Ladybug',170,2178);
-INSERT INTO `mob_skill_lists` VALUES ('Ladybug',170,2179);
-INSERT INTO `mob_skill_lists` VALUES ('Ladybug',170,2180);
-INSERT INTO `mob_skill_lists` VALUES ('Ladybug',170,2181);
+INSERT INTO `mob_skill_lists` VALUES('Ladybug_Day', 170, 2178);
+INSERT INTO `mob_skill_lists` VALUES('Ladybug_Day', 170, 2179);
+INSERT INTO `mob_skill_lists` VALUES('Ladybug_Day', 170, 2180);
+INSERT INTO `mob_skill_lists` VALUES('Ladybug_Day', 170, 2181);
```

Comment:
No need for the extra spaces in here, the rest of the file doesn't seem to use them

Reply:
--amend is a new one for me. I went ahead and got rid of the extra commits and amended them to the last commit. Also fixed this spacing issue.